packages/app/pubspec.yaml

```diff
@@ -31,6 +31,7 @@ dependencies:
   path: 1.9.0
   path_provider: 2.1.3
   shared_preferences: 2.2.2
+  sliver_tools: 0.2.12
```

Comment:
It's basically impossible to do something like `Column` but with slivers inside .. it took me 2 days to find that out 😢 This package and it's `MultiSliver` came to help

Reply:
Oof, thanks for figuring this out, it will be a good pattern to use in other places too.